src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -30,18 +32,17 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+    throw new IllegalArgumentException("numberOfTorpedos");
```

Comment:
You should add tab before `throw` for a more uniform code.

Reply:
Wow, good catch. Thanks, will be fixed :).